### PR TITLE
LPD-39352 Content dashboard filter by results throws Oracle Error

### PR DESCRIPTION
**Pull request by jkappler**
Content dashboard filter by results throws Oracle Error

# What is this trying to solve?

[Link to ticket](https://issues.liferay.com/browse/LPD-39352)

Service builder has an issue where oracle will throw an error when more than 1000 elements are used in a query. In this case we are refering to sites. In content dashboard we retrieve all sites of the virtual instance to then obtain the structures for it, so there can be a too many sites

# How am I fixing it?

Change infoItemFormVariations provider so that we can obtain the elements by companyId instead of all the groups of a given company

# How can you verify that it works?

Run the integration tests, or follow the steps in the ticket

# Why doesn't this include any test?

### CODE REVIEW

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

File: sql/indexes.sql [hunk 1]
```diff
@@ -105,6 +105,7 @@ create index IX_4F40FE5E on DLFileEntryMetadata (fileEntryId);
 create unique index IX_BE290777 on DLFileEntryMetadata (fileVersionId, ctCollectionId, DDMStructureId);
 create index IX_D49AB5D1 on DLFileEntryMetadata (uuid_[$COLUMN_LENGTH:75$]);
 
+create index IX_D2F8189A on DLFileEntryType (companyId);
```
Comment (balazssk) [inline sql/indexes.sql:108]: hey @jkappler ,
Can this somehow include ctCollectionId so that Publications can also leverage this improvement?

**Comment by jkappler** [comment]
ci:stop

**Comment by jkappler** [comment]
@adolfopa, @balazssk mentioned that we need to add the upgrade process for the index

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:report:42257658

**Comment by jkappler** [comment]
Tests are not related

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:report:43070577

**Comment by jkappler** [comment]
ci:forward:force

**Comment by jkappler** [comment]
ci:test:stable

**Comment by jkappler** [comment]
ci:test:stable

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:report:44403796

**Comment by jkappler** [comment]
ci:report:44403796

**Comment by jkappler** [comment]
ci:report:44403796

**Comment by jkappler** [comment]
ci:report:44403796

**Comment by jkappler** [comment]
ci:report:44403796

**Comment by jkappler** [comment]
ci:report:44403796

**Comment by jkappler** [comment]
ci:report:45275447

**Comment by jkappler** [comment]
ci:forward:force

**Comment by jkappler** [comment]
ci:forward:force

**Pull request closed without merging**